modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -145,7 +177,7 @@ private void _createFDSActionObjectDefintion(
 
 		ObjectDefinition fdsActionObjectDefinition =
 			_objectDefinitionLocalService.addSystemObjectDefinition(
-				"FDSAction", userId, 0, "FDSAction", "FDSAction", false,
+				"FDSAction", userId, 0, "FDSAction", null, false,
```

Comment:
This allows object service to decide table name, right?

It's an odd solution, _feels_ like a quick & dirty fix, allowing multiple tables. But I may be necessary, because of the implementation of Objects, tied to portal instances. A more sensible architecture, in my opinion, would be that all Object Entries contain `companyId`, like many standard DXP entities. I don't know if there is a reason why it's not like this for Objects.